### PR TITLE
Added Payeezy Issue #9

### PR DESCRIPTION
**Pull request by hd719**
Hello @kytwb this is my first pull request, I think I added Payeezy (issue #9) correctly to the code. 

-If something is wrong please do let me know and I will fix it asap! 

-Thank you.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ var Omise = require('42-cent-omise').factory;
 var Stripe = require('42-cent-stripe').factory;
 var Braintree = require('42-cent-braintree').factory;
 var PayPal = require('42-cent-paypal').factory;
+var Payeezy = require('42-cent-payeezy').factory
```
Comment (kytwb) [inline index.js]: The module `42-cent-payeezy` doesn't exist.  
The big of the work in adding Payeezy support is to create this module.

Comment (hd719) [inline index.js]: Hey @kytwb so real quick question how would I create this module?

Comment (hd719) [inline index.js]: Woud I have to add `"42-cent-payeezy": "^1.0.0",` in the `package.json` file?

Comment (kytwb) [inline index.js]: You need to create a new repository, write the code for the module, then publish the module on npm. The important part here is `write the code`. Check how the other payment gateways are implemented please.

Comment (hd719) [inline index.js]: I see, I'll do that and send a new pull request.

-Thank you.

**Pull request closed without merging**